types/http.d.ts

```diff
@@ -79,34 +79,34 @@ export declare class HttpRequest {
     /**
      * HTTP request method used to invoke this function.
      */
-    method: HttpMethod;
+    readonly method: string;
```

Comment:
Is it not a downgrade in types for this to be just any string? Does `undici` not enforce its own type on the method?

Reply:
`undici` uses string and I generally trust them to know more about this than us. My hunch is that our previous types were technically incorrect for some reason, for example casing or the fact that there are actually a bunch of possible methods (https://webconcepts.info/concepts/http-method/).